### PR TITLE
Replaces static flight potion moth wings with custom sprite for each wing type

### PR DESCRIPTION
**Pull request by ChesterTheCheesy**
## About The Pull Request
kinda ports [Mothpeople can now use flight potions](https://github.com/yogstation13/Yogstation/pull/11040) from yogs but i made enough changes to call it a new thing

Replaces Mothra and Megamoth wings with custom sprites for every single moth wing type.
obraz

As far as i tested, those wings overlap correctly with different moth bodyparts (fluff and markings).
I wanted to make Mothra and Megamoth into normal wing types, but the sprites are way too big. If you don't want them gone then hook me up with some good replacement sprite.

## Why It's Good For The Game
Makes you not lose your unique moth wings as soon as someone forces you to chug flight potion.
_Might make moths more huggable._

The sprites were made by me, usually based on real moth pictures.

## Changelog
:cl:
add: Added unique spread wing states for every wing type
del: Removed Mothra and Megamoth wings
tweak: Unique moth wings are no longer lost after drinking flight potion
imageadd: Added 22 spreaded wing sprites
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -2175,6 +2182,9 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 /datum/species/proc/CanFly(mob/living/carbon/human/H)
 	if(H.stat || H.body_position == LYING_DOWN)
 		return FALSE
+	if(ismoth(H) && H.dna.features["moth_wings"] == "Burnt Off") //this is so tragic can we get an "F" in the chat
+		to_chat(H, "<span>Your crispy wings won't work anymore!</span>")
```
Comment (ZephyrTFA) [inline code/modules/mob/living/carbon/human/species.dm]: This span does literally nothing.

Comment (ChesterTheCheesy) [reply code/modules/mob/living/carbon/human/species.dm]: you're right!

**Comment by ZephyrTFA** [comment]
Don't forget to credit the original sprite authors

**Comment by ChesterTheCheesy** [comment]
> Don't forget to credit the original sprite authors

Uuh, you mean the original sprites from tg 2016?
I noted the sprites i add i had made myself.

**Comment by ZephyrTFA** [comment]
> > Don't forget to credit the original sprite authors
> 
> Uuh, you mean the original sprites from tg 2016? I noted the sprites i add i had made myself.

don't forget to credit yourself then, smh

**Comment by ChesterTheCheesy** [comment]
updated feathery wings so they are no longer a copypaste of human feather wings and look akin to other mothy wings
obraz

**Comment by Sylphily** [comment]
dude holy shit those wings look sick

**Comment by Latentish** [comment]
Based moth

File: code/modules/mob/living/carbon/human/species.dm [hunk 2]
```diff
@@ -2135,6 +2137,11 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 /datum/species/proc/GiveSpeciesFlight(mob/living/carbon/human/H)
 	if(flying_species) //species that already have flying traits should not work with this proc
 		return
+	if(ismoth(H) && H.dna.features["moth_wings"]) //if we have mothwings, we keep them and not grow new ones
```
Comment (Fikou) [inline code/modules/mob/living/carbon/human/species.dm:2140]: why are you checking if we're a moth? this is givespeciesflight. a species proc. just override it on the moth species.

File: code/modules/mob/living/carbon/human/species.dm [hunk 3]
```diff
@@ -2175,6 +2182,9 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 /datum/species/proc/CanFly(mob/living/carbon/human/H)
 	if(H.stat || H.body_position == LYING_DOWN)
 		return FALSE
+	if(ismoth(H) && H.dna.features["moth_wings"] == "Burnt Off") //this is so tragic can we get an "F" in the chat
+		to_chat(H, "Your crispy wings won't work anymore!")
+		return FALSE
```
Comment (Fikou) [inline code/modules/mob/living/carbon/human/species.dm:2187]: same here

**Review by Fikou — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/species.dm (hunk 2 above)
Comment (ChesterTheCheesy) [reply code/modules/mob/living/carbon/human/species.dm:2140]: it won't work without it, but i should probably add a check for a mothwing organ instead

**Comment by ChesterTheCheesy** [comment]
do not merge until i fix the wing organ code

**Comment by Ketrai** [comment]
These are rad but I feel like they will take up a lot of screen space compared to other wings. Still. That's only a minor annoyance since you can toggle these.

**Comment by Sylphily** [comment]
tbh all the wings need a polish
they also need code that is missing to make them organs

**Comment by ChesterTheCheesy** [comment]
> These are rad but I feel like they will take up a lot of screen space compared to other wings. Still. That's only a minor annoyance since you can toggle these.

All of those are smaller than lizard/megamoth sprites

**Comment by ChesterTheCheesy** [comment]
closing since i'll have to re-factor wings alltogether which will probably take some time

**Pull request closed without merging**